### PR TITLE
Fix GuiEnter_Turkish test

### PR DESCRIPTION
**Pull request by k-takata**
The message will be translated if the Turkish message file is installed.
Catch both English and Turkish messages.

### CODE REVIEW

**Comment by k-takata** [comment]
Failing example:
https://github.com/k-takata/vim-kt/actions/runs/10674485417/job/29585063287#step:20:10003

File: src/testdir/test_autocmd.vim [hunk 1]
```diff
@@ -4871,11 +4871,11 @@ func Test_GuiEnter_Turkish_locale()
...
     lang tr_TR.UTF-8
     let result = execute(':au GuiEnter')
-    call assert_equal("\n--- Autocommands ---", result)
+    call assert_match("\n--- \\(Autocommands\\|Otokomutlar\\) ---", result)
     let result = execute(':au GUIENTER')
-    call assert_equal("\n--- Autocommands ---", result)
+    call assert_match("\n--- \\(Autocommands\\|Otokomutlar\\) ---", result)
     let result = execute(':au guienter')
-    call assert_equal("\n--- Autocommands ---", result)
+    call assert_match("\n--- \\(Autocommands\\|Otokomutlar\\) ---", result)
```
Comment (zeertzjq) [inline src/testdir/test_autocmd.vim]: Maybe `gettext("\n--- Autocommands ---")` can be used for this?

Comment (k-takata) [reply src/testdir/test_autocmd.vim]: Ah, I will try it.

**Comment by k-takata** [comment]
It is strange that Test_autocmd_with_block() is also affected by `:lang`:
https://github.com/k-takata/vim-kt/actions/runs/10674485417/job/29587697797#step:20:10003

**Comment by k-takata** [comment]
It passed in my local CI:
https://github.com/k-takata/vim-kt/actions/runs/10674485417

**Comment by chrisbra** [comment]
thanks.

**Pull request closed without merging**